### PR TITLE
🚧 Add cards and teaser to new Get Started page

### PR DESCRIPTION
**Pull request by lluerich**

### CODE REVIEW

File: pages/content/amp-dev/documentation/index-2021.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    headline: WordPress
+    logo:
+      src: /static/img/documentation/wordpress.png
+      width: 64
+      height: 64
+    text: "All the AMP benefits in the WordPress CMS."
+    url: https://amp-wp.org/documentation/success-with-wordpress/
+  - name: Eleventy
+    headline: 11ty
+    logo:
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/index-2021.html:24]: Can we use SVG icons for those?

**Review by matthiasrohmer — APPROVED** [review]
Thanks so far! Only nits and questions. *Fleißsternchen* for writing meaningful `alt` texts.

File: frontend21/scss/components/content/cards.scss [hunk 2]
```diff
@@ -28,6 +34,14 @@
...
       margin-bottom: 20px;
     }
+
+    &.--stretch-column {
+      display: flex;
+      flex-direction: column;
+      p {
+        flex-grow: 2;
+      }
+    }
```
[the remark this answers is not in the document]
Comment (lluerich) [reply frontend21/scss/components/content/cards.scss:44]: Yes, that would definitely work. I'll change it.

File: pages/content/amp-dev/documentation/index-2021.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+        <div class="ap-m-lnk">
+          <div class="ap-a-ico ap-m-lnk-icon">
+            {% do doc.icons.useIcon('icons/internal.svg') %}
+            <svg>
+              <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#internal"></use>
+            </svg>
+          </div>
+          <span class="ap-m-lnk-text">{{ _('Start with') }} {{ framework.name }}</span>
+        </div>
```
[the remark this answers is not in the document]
Comment (lluerich) [reply pages/content/amp-dev/documentation/index-2021.html:84]: Yes, it definitely makes sense to have this, if you look at the usage of this link.

File: frontend21/scss/components/content/cards.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (lluerich) [reply frontend21/scss/components/content/cards.scss]: Done.

File: pages/content/amp-dev/documentation/index-2021.html (hunk 1 above)
Comment (lluerich) [reply pages/content/amp-dev/documentation/index-2021.html:24]: Ferdi does not have them at this point, but once I have the SVGs, I'm sure we can use those instead.

Comment (lluerich) [reply pages/content/amp-dev/documentation/index-2021.html:24]: PTAL

**Pull request merged**